src/file_upload.rs

```diff
@@ -113,3 +118,30 @@ pub async fn upload_file(
         .append_header((header::LOCATION, return_path))
         .finish())
 }
+
+/// Guarantee that the path is relative and cannot traverse back to parent directories
+/// and optionally prevent traversing hidden directories.
```

Comment:
For security critical stuff like this I often like an illustrating example in the documentation itself to make it entirely clear what kind of attacks are prevented by a given measure. As such, could you please add such an example here? Like which paths are input and what the expected output of this function is.